src/pages/Home.js

```diff
@@ -1,72 +1,160 @@
+      fontSize: '14px',
+      textAlign: 'center',
+      textIndent: '10px',
+      whiteSpace: 'pre-wrap',
+      height: '40px',
+      padding: '10px 0'
+    }
+
+    const paragraph = {
+      color: '#1F3B50',
+      padding: '0 10rem 2rem 10rem',
+      fontFamily: 'Montserrat',
+      fontSize: '14px',
+      textAlign: 'justify',
+      textIndent: '10px',
+      whiteSpace: 'pre-wrap'
+    }
+
+    const buttonStyle = {
+      borderRadius: '5px',
+      backgroundColor: '#00689F',
+      fontFamily: 'Montserrat',
+      fontSize: '16px',
+      fontWeight: 500,
+      color: '#FFFFFF',
+      width: '50%',
+      padding: '10px 20px',
+      margin: '2rem auto'
+    }
 
     return (
 
-      div({ className: 'row home' }, [
-        div({ className: 'col-lg-10 col-lg-offset-1' }, [
-          h1({ className: 'home-title' }, ['Data Use Oversight System']),
-          div({ className: 'home-title-description' },
-            ['Expediting data access for researchers, by facilitating and enhancing data access committees\' workflows']),
-          hr({ className: 'home-line' }),
-          div({ style: { textAlign: 'center' } }, [
-            div({ isRendered: !this.props.isLogged }, [
-              div({ style: { fontSize: 'larger', fontWeight: '500' } }, ['Want to explore datasets and make a data access request?']),
-              div({ style: { margin: '1rem 0 1rem 0' } }, [
-                SignIn({ props: this.props, onSignIn: () => this.props.onSignIn(), history: this.props.history })
-              ]),
-              div({ style: { margin: '0 0 1rem 0' } }, [
-                span({}, [
-                  'Don\'t have a Google Account? Create a new one ',
-                  a({
-                    href: 'https://accounts.google.com/SignUp?continue:https%3A%2F%2Faccounts.google.com%2Fo%2Foauth2%2Fauth%3Fopenid.realm%26scope%3Demail%2Bprofile%2Bopenid%26response_type%3Dpermission%26redirect_uri%3Dstoragerelay%3A%2F%2Fhttp%2Flocalhost%3A8000%3Fid%253Dauth721210%26ss_domain%3Dhttp%3A%2F%2Flocalhost%3A8000%26client_id%3D832251491634-smgc3b2pogqer1mmdrd3hrqic3leof3p.apps.googleusercontent.com%26fetch_basic_profile%3Dtrue%26hl%3Des-419%26from_login%3D1%26as%3D43c5de35a7316d00&ltmpl:popup',
-                    target: '_blank'
-                  }, ['here, ']),
-                  'or sign up with an existing email address ',
-                  a({
-                    href: 'https://support.google.com/accounts/answer/176347',
-                    target: '_blank'
-                  }, ['using these instructions.'])
-                ])
-              ])
+      div({ className: 'row' }, [
+        div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12' }, [
+          div({ className: 'row no-margin', style: { backgroundColor: 'white', height: '350px', position: 'relative' }}, [
+            img({ style: { height: 'inherit', minWidth: '100%', transform: 'scale(1.1,1)', zIndex: '-99' }, src: '/images/home_header_background.png'}),
```

Comment:
So on Firefox the banner image has a zIndex of -99, which puts it behind every element on the page. That's fine except for the fact that its parent div has ```backgroundColor:white```, which will cover up the banner image. Use ```backgroundColor: transaprent``` instead to let the banner show.